packages/chains/src/types.ts

```diff
@@ -47,5 +47,6 @@ export type NativeCurrency = {
 
 export type RpcUrls = {
   http: readonly string[];
+  headers?: object;
```

Comment:
Not convinced by this because headers should be dependant on the specific provider used.